### PR TITLE
Only push national events to the homepage table

### LINKED ISSUE #260: Prevent state leg events from showing on home page table default view

### PR DESCRIPTION
**Pull request by pdvorchik**
Temporary fix to close #260...but the parsing is kinda clunky.  

I'd suggest making `isStateEvent` a bool flag on the model and checking it that way instead.

### CODE REVIEW

File: scripts/models/TownHall.js [hunk 1]
```diff
@@ -111,6 +111,17 @@
     }, data).sort(TownHall.sortFunction);
   };
 
+  // TODO: replace this with a flag on the model from the server
+  TownHall.isStateEvent = function(townhall) {
+    // Some state events don't seem to have the "District" value
```
Comment (meganrm) [inline scripts/models/TownHall.js]: a better check is thp_id, eventually none of the events will have `District`. in the long run I am going to put them at their own endpoint.

Comment (pdvorchik) [reply scripts/models/TownHall.js]: gotcha.  Is it safe to assume if an object does not have a thp_id then it's not a state event for now?

Comment (meganrm) [reply scripts/models/TownHall.js]: yeah

**Comment by meganrm** [comment]
I moved the state events to their own enpdoint

**Pull request closed without merging**